libdrgn/drgn.h.in

```diff
@@ -637,6 +645,32 @@ static inline size_t drgn_type_num_members(struct drgn_type *type)
 	return type->_private.num_members;
 }
 
+/**
+ * Get whether a type has template parameters.
+ */
+static inline bool drgn_type_has_template_parameters(struct drgn_type *type)
+{
+	return type->_private.num_template_parameters > 0;
+}
+
+/**
+ * Get the template parameters of a type.
+ */
+static inline struct drgn_template_parameter *drgn_type_template_parameters(struct drgn_type *type)
+{
```

Comment:
This should `assert(drgn_type_has_template_parameters(type))` like the other getters.